applications/nrf_desktop/README.rst

```diff
@@ -2040,20 +2040,34 @@ Image transfer over SMP
-The `nRF Connect for Mobile`_ application uses binary files for the image transfer over the Simple Management Protocol (SMP).
-The :guilabel:`DFU` button appears in the tab of the connected Bluetooth device that supports the image transfer over the SMP.
-After pressing the button, you can select the :file:`*.bin` file used for the firmware update.
+The `nRF Connect Device Manager`_ application uses binary files for the image transfer over the Simple Management Protocol (SMP).
 
-After building your application for configuration with either the :ref:`nrf_desktop_ble_smp` or :ref:`nrf_desktop_dfu_mcumgr` enabled, the following firmware update files are generated in the build directory:
+After building your application for configuration with the |smp| enabled, the :file:`dfu_application.zip` archive is generated in the build directory.
+It contains all necessary firmware update files to perform the DFU.
+See :ref:`app_build_fota` for more information about the contents of update archive.
 
- * :file:`zephyr/app_update.bin` - The application image that is bootable from the primary slot.
- * :file:`zephyr/mcuboot_secondary_app_update.bin` - The application image that is bootable from the secondary slot.
-   The file is generated only if the MCUboot bootloader is built in the direct-xip mode.
+See how you can perform the DFU using the `nRF Connect Device Manager`_ mobile app:
 
-If the MCUboot bootloader uses the built-in direct-xip mode, you must upload the image for the slot that is currently unused.
-If the MCUboot bootloader built-in swap mode is used, the :file:`zephyr/app_update.bin` file must always be used.
-The :file:`zephyr/app_update.bin` file is the application image that can be booted from the primary slot.
-In the swap mode, the MCUboot bootloader always moves the new application image to the primary slot before booting it.
+1. Download the :file:`dfu_application.zip` archive to your device.
+#. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
+
+  a. Ensure that you can access the :file:`dfu_application.zip` archive from your phone or tablet.
+  #. In the mobile app, scan and select the device to update.
+  #. Switch to the :guilabel:`Image` tab.
+  #. Tap the :guilabel:`SELECT FILE` button and select the :file:`dfu_application.zip` archive.
+  #. Tap the :guilabel:`START` button.
+  #. Initiate the DFU process of transferring the image to the device:
+
+    * If you are using an Android device, select a mode in the dialog window, and tap the :guilabel:`START` button.
+    * If you are using an iOS device, tap the selected mode in the pop-up window.
+
+  #. Wait for the DFU to finish and then verify that the application works properly.
+
+.. note::
+  Support for FOTA updates with MCUboot in the direct-xip mode is available since the following versions of the `nRF Connect Device Manager`_ mobile app:
+
+  * Version ``1.8.0`` on Android.
+  * Version ``1.4.0`` on iOS.
```

Comment:
Similarly, you can define a label for the following note

(also applies to CAF rst)